runtime/common/src/claims.rs

```diff
@@ -292,6 +302,10 @@ impl<T: Trait> sp_runtime::traits::ValidateUnsigned for Module<T> {
 		const PRIORITY: u64 = 100;
 
 		match call {
+			// <weight>
+			// Base Weight: 370 µs
+			// DB Weight: 1 Read (Claims)
+			// </weight>
```

Comment:
So validate_unsigned time computation is not part of claim weight because it less than ExtrinsicBaseWeight ?

Reply:
Yes, and we noted that in our Riot chat. I think this is supposed to be "good enough for now".

I think though, to be safe, I should include this weight into the main extrinsic until we polish all the details.

Reply:
yes, IMHO currently ExtrinsicBaseWeight to be 10µs so I would say better to put into the extrinsic as well. In the future maybe we should have SignedExtrinsicBaseWeight UnsignedExtrinsicBaseWeight.

(Also the benchmark could include both so read/writes are not duplicated (but here its fine))

Reply:
> In the future maybe we should have SignedExtrinsicBaseWeight UnsignedExtrinsicBaseWeight.

yes, I think that will be the case